test/e2e/domainmapping/domain_mapping_test.go

```diff
@@ -83,6 +84,9 @@ func TestBYOCertificate(t *testing.T) {
 	secret, err := clients.KubeClient.CoreV1().Secrets(test.ServingFlags.TestNamespace).Create(ctx, &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: test.AppendRandomString("byocert-secret"),
+			Labels: map[string]string{
+				networking.CertificateUIDLabelKey: "byocert-secret",
+			},
```

Comment:
Is this related to the internal encryption?
I think this label is not necessary for the internal encryption at least.

Reply:
I had to add it for net-istio too. It is not about internal-encryption but it is needed for tests to pass anyway.

Reply:
Please add this label when you enable the filtering for test. Currently the tests does not enable filtering even with merging this PR, does it?

Reply:
True but if someone wants to run the suite with the env enabled? I can image a scenario where we test this downstream for example.

Reply:
What I am concerned is that it will hide whether we always need to add the label to DomainMapping or not.

Could you please open a separated PR if you want to add it? I think I can approve the internal-encryption stuff, but this DomainMapping label should be reviewed by others reviewers.

Reply:
Ok sure.